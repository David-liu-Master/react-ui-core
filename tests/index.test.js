import React from 'react';
import { create as renderer } from 'react-test-renderer';
import createTestContext from 'react-cosmos-test/generic';
import fixture from '../fixtures/index.fixture';

const { mount, getWrapper } = createTestContext({ renderer, fixture });

beforeEach(mount);

test('renders index', () => {
  expect(getWrapper().toJSON()).toMatchSnapshot();
});
