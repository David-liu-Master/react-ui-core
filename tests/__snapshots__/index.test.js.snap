// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders index 1`] = `
<div>
  <h2>
    Welcome to React components
  </h2>
  <button
    className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-text-3 MuiButton-flat-6"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-2"
    >
      Click Me
    </span>
    <span
      className="MuiTouchRipple-root-30"
    />
  </button>
</div>
`;
